docs/compiling.rst

```diff
@@ -68,6 +68,23 @@ that is supported via a ``build_ext`` command override; it will only affect
         ext_modules=ext_modules
     )
 
+Since pybind11 does not require NumPy when building, a light-weight replacement
+for NumPy's parallel compilation distutils tool is included. Use it like this:
+
+    from pybind11.setup_helpers import ParallelCompile
+
+    # Optional multithreaded build
+    ParallelCompile("NPY_NUM_BUILD_JOBS").install()
```

Comment:
Isn't it then still a bit weird to refer to a `NPY` environment variable?

Reply:
It's best to show a variable here (it is optional, but if you don't supply it, there's no way to control the number of threads manually, which might be nice to have). Since NumPy uses this, I just chose it as a standard (and it makes it a true drop in replacement).

Reply:
(I'd be open to alternative suggestions for this name, though; it's just an example)

Reply:
> Since NumPy uses this, I just chose it as a standard (and it makes it a true drop in replacement).

I see where you're coming from, yes. It's just a bit confusing at first sight.